cornflow/tests/unit/test_cases.py

```diff
@@ -307,3 +314,66 @@ def test_case_does_not_exist(self):
         )
         self.assertEqual(response.status_code, 404)
         self.assertEqual(response.json["error"], "The object does not exist")
+
+class TestCaseJsonPatch(CustomTestCase):
+    def setUp(self):
+        super().setUp()
+        self.payload = self.load_file(CASE_PATH)
+        self.model = CaseModel
+        self.case_id = self.create_new_row(CASE_URL, self.model, self.payload)
+        self.payloads = [self.load_file(f) for f in CASES_LIST]
+        self.items_to_check = ["name", "description", "path", "schema"]
+        self.url = CASE_URL
+        self.patch = {
```

Comment:
we're validating jsonpatch with jsonpatch? I think we should put the difference inside explicitly in the test so we know what we're expecting as a json. It should not be too big (we can generate it once, validate it and paste it as json/dict)

Reply:
You are right, i should have a json-patch file on the data folder and test that json-patch creates that file and if we send that file it works.

I think we can keep these test nonetheless and create new ones reading the json-patch from a file